marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mappingWithCustomHeader.cy.tsx

```diff
@@ -139,7 +139,12 @@ describe("Create and verify load steps, map step and flows with a custom header"
     cy.waitForAsyncRequest();
 
     cy.log("**Open Order to see steps**");
-    curatePage.getEntityTypePanel("Order").should("be.visible").click({force: true});
+    curatePage.getEntityTypePanel("Order").then(($ele) => {
```

Comment:
Could we add the accordion logic to the getEntityTypePanel? That way we don't need to repeat it in other places.